openstack_controller/tests/fixtures/GET/compute/v2.1/os-hypervisors/detail/limit=1/response.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
`limit` argument was added in version `X-OpenStack-Nova-API-Version` 2.33 as you can see [here](https://docs.openstack.org/api-ref/compute/#list-hypervisors-details)
Before that version the argument was ignored and the entire list was returned

Reply:
good catch!